usl_pipeline/study_area_uploader/study_area_uploader/main.py

```diff
@@ -167,5 +168,12 @@ def parse_args() -> argparse.Namespace:
         + "console (INFO logging level instead of default ERROR one)",
         action=argparse.BooleanOptionalAction,
     )
+    parser.add_argument(
+        "--non-green-area-soil-classes",
```

Comment:
Is this something callers should pass in? Or is this just something we should hard-code?
Is this actually something that will vary across study areas?
My concern would be that callers would have to supply the same value to this script every time to get correct behaviour, and I'm not sure if it's actually something callers need or want to vary.

Reply:
Yeah, I think soil classes are very region specific. I'm not sure there are unified codes recognized by globally.

Reply:
We can't. Soil classes as a whole is an optional data source.

Reply:
Could we make it conditionally required if soil classes are present?

Reply:
Done